<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyboardDockTitleWidget</class>
 <widget class="QWidget" name="KeyboardDockTitleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>31</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>-1</number>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Keyboard Monitor</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KSeparator" name="kseparator">
     <property name="frameShape">
      <enum>QFrame::VLine</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="sizeSlider">
     <property name="minimum">
      <number>36</number>
     </property>
     <property name="maximum">
      <number>150</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KSeparator</class>
   <extends>QFrame</extends>
   <header>kseparator.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
